Descriptors/ProgramTypeDescriptor.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<InterchangeDescriptors xmlns="http://ed-fi.org/3.3.1-b" xmlns:ann="http://ed-fi.org/annotation" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://ed-fi.org/3.3.1-b ../Schemas/Bulk/EXTENSION-Interchange-Descriptors-Extension.xsd">
+	<ProgramTypeDescriptor>
+		<CodeValue>Traditional</CodeValue>
```

Comment:
Since this is on ProgramTypeDescriptor which is used in different contexts in core, it might make sense to have the CodeValue and ShortDescription for this one match the Description so it's clear.

Same for the others below.